examples/custom_metrics/README.md

```diff
@@ -0,0 +1,100 @@
+# Monitoring Torchserve custom metrics with mtail metrics exporter and prometheus
+
+In this example, we show how to use a pre-trained custom MNIST model and export the custom metrics using mtail and prometheus
+
+We used the following pytorch example to train the basic MNIST model for digit recognition : https://github.com/pytorch/examples/tree/master/mnist
+
+Run the commands given in following steps from the parent directory of the root of the repository. For example, if you cloned the repository into /home/my_path/serve, run the steps from /home/my_path
+
+## Steps
+
+- Step 1: By default prediction time and handler time are the two custom metrics added in the base handler.
+New metrics can be added to handler in the handle method. In this example we export the `HandlerTime` and `PredictionTime` using mtail.
+
+  ```python
+  def handle(self, data, context):
```

Comment:
Please add custom metric in the custom handler: https://github.com/pytorch/serve/blob/master/docs/metrics.md#log-custom-metrics